env.md

```diff
@@ -0,0 +1,118 @@
+# Inventory of Agoric SDK Environment Variables
+
+```js
+process.env.DOCUMENTATION // for power users
+```
+
+TODO: Fill out more details!
```

Comment:
Is there a specific detail that we know is not filled out? If not, this seems superfluous. Our documentation is always best-effort, no?